README.md

```diff
@@ -1,7 +1,9 @@
 tqdm
 ====
 
-Instantly make your loops show a progress meter - just wrap any iterator with "tqdm(iterator)", and you're done!
+[![Build Status](https://img.shields.io/travis/kmike/tqdm.svg?branch=all-fixes)](https://travis-ci.org/kmike/tqdm)
```

Comment:
After merging this line should be changed.

1. Sign in to https://travis-ci.org/, go to your Account settings and turn ON the switch near tqdm repo;
2. Change this line to show a badge from the official repo: 

```
[![Build Status](https://img.shields.io/travis/noamraph/tqdm.svg)](https://travis-ci.org/noamraph/tqdm)
```